manifests/t/TeamRadarr/Radarr/4.5.2.7388/TeamRadarr.Radarr.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 4.5.2.7388
+MinimumOSVersion: 10.0.0.0
+AppsAndFeaturesEntries:
+- DisplayName: Radarr
+  Publisher: Team Radarr
+  DisplayVersion: 4.5.2.7388

```

Comment:
```suggestion
```
Publisher is not needed as it matches the defaultLocale file
DisplayVersion is inaccurate